src/test/resources/localstack/setup-sns.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+export TERM=ansi
+export AWS_ACCESS_KEY_ID=foobar
+export AWS_SECRET_ACCESS_KEY=foobar
+export AWS_DEFAULT_REGION=eu-west-2
+aws --endpoint-url=http://localhost:4575 sns create-topic --name offender_events
```

Comment:
not used in the tests yet, but here so that localstack starts with everything we need